### PR TITLE
Adding third container for jasmine tests to avoid flaky fails

### LINKED ISSUE #3587: Find way to make flaky tests less flaky on CI
Since about a week ago, the `test-jasmine` CI test container has been failing roughly (my estimate) 1 out of two runs. Of those failure most are due to 

```
retry npm run test-jasmine -- --tags=flaky --skip-tags=noCI
```

We should try to find a solution.

### Potential solutions

1) split off that 

```
retry npm run test-jasmine -- --tags=flaky --skip-tags=noCI
```

into a new test container (named `test-jasmine3`)

2) use [`shard_jasmine_tests.js`](https://github.com/plotly/plotly.js/blob/master/tasks/shard_jasmine_tests.js) util as in our `@gl` tests:

https://github.com/plotly/plotly.js/blob/4b21575af725c8f52e0522cc1a9b99778bd6606c/.circleci/test.sh#L57-L61

to split off the `@flaky` tests into multiple runs. 

3) combine 1) and 2)

---------

@antoinerg @archmoj wanna give this a shot?

### PR DESCRIPTION
**Pull request by archmoj**
Fixes #3587.
@plotly/plotly_js

### CODE REVIEW

**Comment by etpinard** [comment]
Have you tried re-running the tests a few times to see if it actually helps?

**Comment by archmoj** [comment]
> Have you tried re-running the tests a few times to see if it actually helps?

Good call. It quickly passed the very first time; but then I see it may not pass.
So I will try the second step described here: https://github.com/plotly/plotly.js/issues/3587.

File: .circleci/test.sh [hunk 1]
```diff
@@ -63,6 +62,18 @@ case $1 in
         exit $EXIT_STATE
         ;;
 
+    jasmine3)
+        set_tz
+
+        SHARDS=($(node $ROOT/tasks/shard_jasmine_tests.js --tag=flaky))
+
+        for s in ${SHARDS[@]}; do
+            retry npm run test-jasmine -- --tags=flaky --skip-tags=noCI
```
Comment (etpinard) [inline .circleci/test.sh]: This line should be:

```
 retry npm run test-jasmine -- "$s"  --tags=flaky --skip-tags=noCI
```

**Comment by etpinard** [comment]
Any updates on this thing?

Even if the tests in `test-jasmine3` (still) fail from time to time, this PR will make our rebuild faster as we won't have to re-test all the non-flaky jasmine tests when we have to rebuild.

**Comment by archmoj** [comment]
Yes. I think it is ready to go.

**Comment by etpinard** [comment]
:dancer:

**Comment by antoinerg** [comment]
Looks good to me :dancer:

**Pull request merged**